clients/java/client/src/main/java/org/camunda/bpm/client/impl/EngineClient.java

```diff
@@ -22,11 +22,7 @@
 
 import org.camunda.bpm.client.task.ExternalTask;
 import org.camunda.bpm.client.task.impl.ExternalTaskImpl;
-import org.camunda.bpm.client.task.impl.dto.BpmnErrorRequestDto;
-import org.camunda.bpm.client.task.impl.dto.CompleteRequestDto;
-import org.camunda.bpm.client.task.impl.dto.ExtendLockRequestDto;
-import org.camunda.bpm.client.task.impl.dto.FailureRequestDto;
-import org.camunda.bpm.client.task.impl.dto.LockRequestDto;
+import org.camunda.bpm.client.task.impl.dto.*;
```

Comment:
Please don't use star imports. This is against our [Java Coding Style Guide](https://github.com/camunda/camunda-bpm-platform/wiki/Coding-Style-Java).

Reply:
thank. fixed